deepchem/models/__init__.py

```diff
@@ -16,8 +16,8 @@
 from deepchem.models.tensorgraph.IRV import TensorflowMultiTaskIRVClassifier
 from deepchem.models.tensorgraph.robust_multitask import RobustMultitaskClassifier
 from deepchem.models.tensorgraph.robust_multitask import RobustMultitaskRegressor
-from deepchem.models.tensorgraph.progressive_multitask import ProgressiveMultitaskRegressor, ProgressiveMultitaskClassifier
-from deepchem.models.tensorgraph.models.graph_models import WeaveTensorGraph, DTNNTensorGraph, DAGTensorGraph, GraphConvModel, MPNNTensorGraph
+from deepchem.models.tensorgraph.progressive_multitask import ProgressiveMultitaskRegressor
```

Comment:
Sorry, missed this on first pass. Could you add the `ProgressiveMultitaskClassifier` import back in?

Reply:
Done and pushed -- sorry about that. I suspect it was a stray keystroke as I was editing on a plane.